doc/index.rst

```diff
@@ -18,6 +18,12 @@ Welcome to the official polybar documentation.
 
    man/polybar.1
 
+.. toctree::
+   :maxdepth: 1
+   :caption: For Developers:
```

Comment:
Maybe
```suggestion
   :caption: For packagers:
```

Reply:
Yeah, I wasn't sure how to categorize this. It isn't really user documentation but also not quite developer documentation. And this will probably be the only page about packaging polybar, so it shouldn't have its own category. Maybe we can name it contributor documentation.

Reply:
> Maybe we can name it contributor documentation.

Make sense.